src/aragon-mock.js

```diff
@@ -0,0 +1,113 @@
+import { apps } from './demo-state'
+
+export class IFrameMessenger {
+  constructor(iframe) {
+    this.iframe = iframe
+  }
+  send = data => {
+    this.iframe.contentWindow.postMessage(data, '*')
+  }
+}
+
+export class WorkerMessenger {
+  constructor(worker) {
+    this.worker = worker
+  }
+  send = data => {
+    this.worker.postMessage(data)
```

Comment:
@onbjerg Note that a worker's `postMessage()` is slightly different from a window's: windows accept a string target origin as their second argument but workers accept an "transferrable" object (usually an `ArrayBuffer`).